### ISSUE TITLE
Add description to renderers-API

### ISSUE THREAD

**Issue by ManuelHentschel** [nx10/httpgd#81]
Would it be feasible to include a short description in the data returned by `/renderers`? E.g. the Renderer column from [here](https://github.com/nx10/httpgd/blob/master/docs/api-documentation.md#get-renderers). This could be useful as hover/description in an `Export plot as...` dialogue.

**Comment by nx10** [comment]
Yes I was also thinking about this, but I think we should keep localization in mind for the implementation. Ideally all user-visible strings in the client (and server messages) should be translatable in the future.

**Comment by ManuelHentschel** [comment]
There are `Accept-Language` headers etc. for this in http, do these also work with websockets?
